docs/source/cpp/compute.rst

```diff
@@ -851,6 +851,8 @@ String transforms
 +-------------------------+--------+-----------------------------------------+------------------------+-----------------------------------+-------+
 | binary_replace_slice    | Unary  | String-like                             | Binary- or String-like | :struct:`ReplaceSliceOptions`     | \(5)  |
 +-------------------------+--------+-----------------------------------------+------------------------+-----------------------------------+-------+
+| binary_reverse          | Unary  | Binary                                  | Binary                 |                                   | \(11) |
```

Comment:
Hmm... can you renumber the notes so that they appear in ascending order in this table? It's a bit cumbersome, but doable :-)